src/app/view/components/listing-data-table/listing-data-table.components.spec.ts

```diff
@@ -358,4 +358,17 @@ describe('ListingDataTableComponent', () => {
         expect(comp.dataTable.first).toBe(1);
         expect(comp.items.length).toBe(7);
     });
+
+    it('should focus first row on arrowDown in Global Search Input', () => {
+        spyOn(comp, 'focusFirstRow').and.callThrough();
+        spyOn(this.paginatorService, 'get').and.returnValue(observableOf(this.items));
+        comp.columns = this.columns;
+        comp.loadFirstPage();
+        fixture.detectChanges();
+        comp.globalSearch.nativeElement.dispatchEvent(new KeyboardEvent('keydown', { key: 'arrowDown' }));
+
+        expect(comp.focusFirstRow).toHaveBeenCalledTimes(1);
```

Comment:
You don't need this, you just need the next expect, because is what you want to do.